### ISSUE TITLE
PaginatedList totalCount raises AttributeError

### ISSUE THREAD

**Issue by jakirkham** [PyGithub/PyGithub#487]
Ran into this rather odd issue locally. Not quite sure what is going on, but figured it deserved an issue as I saw it repeatedly.

```python
>>> l
<github.PaginatedList.PaginatedList instance at 0x...>
>>> l[0]
>>> IssueComment(user=NamedUser(login="..."), id=...)
>>> l.totalCount
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
<ipython-input-34-ba47f831ae47> in <module>()
----> 1 l.totalCount

/zopt/conda2/lib/python2.7/site-packages/github/PaginatedList.pyc in totalCount(self)
    125     def totalCount(self):
    126         if not self.__totalCount:
--> 127             self._grow()
    128 
    129         return self.__totalCount

/zopt/conda2/lib/python2.7/site-packages/github/PaginatedList.pyc in _grow(self)
     59 
     60     def _grow(self):
---> 61         newElements = self._fetchNextPage()
     62         self.__elements += newElements
     63         return newElements

/zopt/conda2/lib/python2.7/site-packages/github/PaginatedList.pyc in _fetchNextPage(self)
    160             self.__nextUrl,
    161             parameters=self.__nextParams,
--> 162             headers=self.__headers
    163         )
    164         data = data if data else []

/zopt/conda2/lib/python2.7/site-packages/github/Requester.pyc in requestJsonAndCheck(self, verb, url, parameters, headers, input, cnx)
    170 
    171     def requestJsonAndCheck(self, verb, url, parameters=None, headers=None, input=None, cnx=None):
--> 172         return self.__check(*self.requestJson(verb, url, parameters, headers, input, cnx))
    173 
    174     def requestMultipartAndCheck(self, verb, url, parameters=None, headers=None, input=None):

/zopt/conda2/lib/python2.7/site-packages/github/Requester.pyc in requestJson(self, verb, url, parameters, headers, input, cnx)
    211             return "application/json", json.dumps(input)
    212 
--> 213         return self.__requestEncode(cnx, verb, url, parameters, headers, input, encode)
    214 
    215     def requestMultipart(self, verb, url, parameters=None, headers=None, input=None):

/zopt/conda2/lib/python2.7/site-packages/github/Requester.pyc in __requestEncode(self, cnx, verb, url, parameters, requestHeaders, input, encode)
    241             requestHeaders["Accept"] = "application/vnd.github.moondragon+json"
    242 
--> 243         url = self.__makeAbsoluteUrl(url)
    244         url = self.__addParametersToUrl(url, parameters)
    245 

/zopt/conda2/lib/python2.7/site-packages/github/Requester.pyc in __makeAbsoluteUrl(self, url)
    302         # URLs generated locally will be relative to __base_url
    303         # URLs returned from the server will start with __base_url
--> 304         if url.startswith("/"):
    305             url = self.__prefix + url
    306         else:

AttributeError: 'NoneType' object has no attribute 'startswith'
```

Using version `1.29` on Python `2.7`.

**Comment by rapha-opensource** [comment]
I'm seeing the same error using version 1.27 with Python3.5.

gh = Github(...,...)
user = gh.get_user(...)
repos = user.get_repos()
repo = repos[0]
repos.totalCount
>>> repos.totalCount
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/venv/rq/lib/python3.5/site-packages/github/PaginatedList.py", line 127, in totalCount
    self._grow()
  File "/venv/rq/lib/python3.5/site-packages/github/PaginatedList.py", line 61, in _grow
    newElements = self._fetchNextPage()
  File "/venv/rq/lib/python3.5/site-packages/github/PaginatedList.py", line 162, in _fetchNextPage
    headers=self.__headers
  File "/venv/rq/lib/python3.5/site-packages/github/Requester.py", line 172, in requestJsonAndCheck
    return self.__check(*self.requestJson(verb, url, parameters, headers, input, cnx))
  File "/venv/rq/lib/python3.5/site-packages/github/Requester.py", line 213, in requestJson
    return self.__requestEncode(cnx, verb, url, parameters, headers, input, encode)
  File "/venv/rq/lib/python3.5/site-packages/github/Requester.py", line 243, in __requestEncode
    url = self.__makeAbsoluteUrl(url)
  File "/venv/rq/lib/python3.5/site-packages/github/Requester.py", line 304, in __makeAbsoluteUrl
    if url.startswith("/"):
AttributeError: 'NoneType' object has no attribute 'startswith'

digging deeper, repos._PaginatedList__nextUrl is 'None'.

**Comment by jakirkham** [comment]
This a bug people are seeing. Why is it being ignored and marked as stale?

**Comment by sfdye** [comment]
@jakirkham please understand this is just stale bot's default behavior to mark issues > 2 months old as stale, will remove the label if it's still an issue.

**Comment by jakirkham** [comment]
Have noticed this bot popping up in more issues on different repos. Thus far have seen it cause more than a few undesirable interactions. Namely closing issues that have not gotten attention from the project's core developer team (often simply because they are busy, which is ok). As a user interested in an OSS project's improvement, I find the bot's behavior in these cases discouraging. FWIW it seems like there is a similar discussion happening over with the stale bot's developer team, which outlines this problem well.

xref: https://github.com/probot/stale/issues/67

**Comment by sfdye** [comment]
@jakirkham Thanks for sharing this. Going back to the issue itself, please see the latest development #820, as we are trying to fix this.

**Comment by sfdye** [comment]
Fixed in #820
